src/structures/Message.js

```diff
@@ -396,7 +410,7 @@ class Message extends Base {
    * @readonly
    */
   get thread() {
-    return this.channel.threads.resolve(this.id);
+    return this.channel?.threads.resolve(this.id) ?? null;
```

Comment:
https://github.com/discordjs/discord.js/pull/6271/files#r681339770

if it can never be null then ...?
```suggestion
    return this.channel.threads.resolve(this.id);
```

Reply:
fully documenting and typing it as nullable is not feasibly possible with how the library is designed at the moment, yet decided to handle it when easily possible, which is the case here, since the return value was already nullable.